soc/arm/ti_simplelink/cc32xx/Kconfig.soc

```diff
@@ -21,3 +25,11 @@ config CC3220SF_DEBUG
 	default XIP
 
 endif # SOC_CC3220SF
+
+if SOC_CC3235SF
+
+config CC3235SF_DEBUG
```

Comment:
Is this the right option name? Wouldn't a more specific, like "CC3235SF_DEBUG_IMG_HDR" would be better?